client/transport/Cargo.toml

```diff
@@ -42,7 +42,9 @@ gloo-net = { version = "0.5.0", default-features = false, features = ["json", "w
 futures-channel = { version = "0.3.14", default-features = false, optional = true }
 
 [features]
-tls = ["tokio-rustls", "rustls-pki-types", "rustls-platform-verifier", "rustls"]
+tls = ["rustls", "tokio-rustls", "rustls-pki-types"]
+tls-rustls-platform-verifier = ["tls", "rustls-platform-verifier"]
+default = ["tls-rustls-platform-verifier"]
```

Comment:
remove

Reply:
Oh, I made some mistake while rebasing for local tests, added your suggestions again.